### DISCUSSION TITLE
What is currently considered to be the most stable version?

### DISCUSSION THREAD

**Discussion by M-Pixel** [teableio/teable#967] [Q&A] [answered]
It seems that at some point the 1.3 releases switched from being labeled "beta" to "alpha".  I'm not sure if this was just a change in policy regarding what to call it, or if it indicates a decrease in stability (despite the lack of an increase in the patch version).  On GitHub, I see 1.4.1 listed as the first and only actual "release", and there's a tag on the container registry "[1.4.1-build.537-amd64](https://github.com/orgs/teableio/packages/container/teable/272406119?tag=1.4.1-build.537-amd64)" that's missing the "alpha" qualifier compared to the ones around it, but there's no plain "1.4.1", leading me to wonder if they're all "alpha" and the aforementioned tag was a fluke.

**Comment by tea-artist** [comment, marked as answer]
We recommend always using the latest version.

We removed the beta suffix in the 1.3.1 version, But there is no concept of "the most stable version" at present. Because stability is usually for different functional modules. And we are still continuing to fix bugs in the core functions.
